05/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/global.css">
+    <link rel="stylesheet" href="./styles/mobile.css">
+    <link rel="stylesheet" href="./styles/desktop.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="nav-container">
+            <input type="checkbox" id="menu-hamburger">
+            <label for="menu-hamburger" class="hamburger">&#9776;</label>
+            <nav >
+                <ul class="menu">
+                    <li class="menu-item">HOME</li>
+                    <li class="menu-item">PRODUCTS
+                        <ul class="submenu">
+                            <li class="submenu-item">FRUITS</li>
+                            <li class="submenu-item">NUTS</li>
+                        </ul>
+                    </li>
+                    <li class="menu-item">CONTACT</li>
+                </ul>
+            </nav>
+            
+        </div>
+    </header>
+
+
+    <section class="section-double">
+        <div class="section-double--item">.</div>
```

Comment:
Jeśli elemet nadrzędny to `section-double` (czyli Block) to dzieci powinny zawierać nazwę jako Element czyli `section-double__item`, a nie Modyfikator. Proponuję jeszcze doczytać: https://devmentor.pl/b/metodologia-bem-w-css-i-sass